app/experimenter/nimbus-ui/src/lib/constants.ts

```diff
@@ -14,8 +14,6 @@ export const SUBMIT_ERROR =
 export const EXTERNAL_URLS = {
   TRAINING_AND_PLANNING_DOC:
     "https://mana.mozilla.org/wiki/display/FJT/Nimbus+Onboarding",
-  RISK_MITIGATION_TEMPLATE_DOC:
```

Comment:
I haven't looked into the other Risk Mitigation issues yet, but do we have a new version of this doc somewhere? Wondering if this link will still be useful

Reply:
You've seen this by now, but we do have [new links](https://github.com/mozilla/experimenter/pull/5170/files#diff-52ccc6c28c664b4fad9c3a9af82889cc7a44bff31bfad6e4b94d1aa47c2825daR29-R34). I'll double check with Ana that we shouldn't keep this one around.